app/src/main/java/com/globant/weatherapp/mvp/view/WeatherView.kt

```diff
@@ -27,4 +29,10 @@ class WeatherView(activity: Activity) : ActivityView(activity), WeatherContracts
             visibility = View.VISIBLE
         }
     }
+
+    override fun showError(){
+        activity?.progress_bar?.visibility = View.GONE
+        Toast.makeText(this.context, R.string.error_message_internet, Toast.LENGTH_SHORT).show()
+        activity?.onBackPressed()
```

Comment:
why do you do this action?

Reply:
To go back to the previous activity, because if the service gives some error, this activity would be empty until the user push the back button on the device.